### ISSUE TITLE
Should channels bleed into the execute command based on condarc?

### ISSUE THREAD

**Issue by pelson** [pelson/conda-execute#30]
It leaves an unpalatable irreproducibility about it, but is also somewhat convenient to avoid having to type `defaults` all the time.

One option: only include those from condarc if no channel directive is set in the execute heaader.

**Comment by ericdill** [comment]
Is there any thought of simply delegating to `conda-env` and making the `conda-execute` header match the `environment.yaml` spec?

**Comment by pelson** [comment]
> Is there any thought of simply delegating to conda-env and making the conda-execute header match the environment.yaml spec?

Yep. I'd be game for that. Obviously conda-env doesn't do *all* of it (e.g. `run_with`, installing into a managed temporary environment), but the bits it can do (particularly the env definition) I'm up for using it.
